bittensor/dendrite.py

```diff
@@ -164,7 +164,10 @@ async def aclose_session(self):
         """
         if self._session:
             await self._session.close()
-            self._session = None
+            try:
+                self._session = None
+            except:
+                pass
```

Comment:
Rather than `except` then `pass`, should we have some logging to see what the error would be, just in case?

Reply:
Yep. Added.